vanilla-framework/vanilla-framework
# 1441: Remove border-bottom on last row of table

## Done

- Removed border-bottom on last row of table by making it transparent (as opposed to removing entirely)

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Check that the base Table, Table expandable, Table sortable and Table mobile card all have the last row border removed

## Details

Fixes #1441
